hack/update-bazel.sh

```diff
@@ -22,5 +22,8 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 git config http.https://gopkg.in.followRedirects true
 
+# Remove generated files prior to running gazel.
+rm -f "${KUBE_ROOT}/pkg/generated/openapi/zz_generated.openapi.go"
```

Comment:
Added a TODO. Our generated code has the `!ignore_autogenerated` tag on it that we might be able to use, but only once bazel can generate all of our code.